components/symbl_ai/actions/submit-video-url/post-video-url.js

```diff
@@ -0,0 +1,52 @@
+import axios from "axios";
```

Comment:
If you modify the file `symbl_ai.app.mjs` as I suggested here, then you can replace this line with
```suggestion
import symblAIApp from "../../symbl_ai.app.mjs";
```